libs/virtual_joystick/joystick.kv

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) farm-ng, inc.
```

Comment:
I’d advocate to put kivy files under src/res

Reply:
:+1: didn't see that from the template until editing dev-kit docs

Reply:
Oh I thought you meant for main... I'd like to be able to import the joystick Widget 's entirety in from lib